console/__init__.py

```diff
@@ -32,8 +32,8 @@
     ftp = FTP(settings.FTP_HOST)
     ftp.login(user=settings.FTP_USER, passwd=settings.FTP_PASS)
     len([fname for fname, fmeta in ftp.mlsd(path=settings.SDX_FTP_DATA_PATH)])
-except Exception as e:
-    logger.info("MLSD command not availible in the FTP server, will use LIST instead", exception=e)
+except Exception:
+    logger.exception("MLSD command not availible in the FTP server, will use LIST instead")
```

Comment:
Why remove the exception?  Will it not be useful in finding issues.

Reply:
@owencjones `logger.exception(...` includes a stacktrace so the variable isn't required 👍